documentation/src/main/asciidoc/topics/code_examples/HotRodMetrics.java

```diff
@@ -0,0 +1,7 @@
+HotRodMetricRegistry registry = ...;
```

Comment:
Maybe I'm just missing it, but we should have an example in the docs of how to create the registry.

Reply:
It is in the file below

Reply:
Well that's embarrassing :rofl:

Reply:
Well that's embarrassing! :rofl: